README.md

```diff
@@ -66,12 +79,40 @@ targets: [
 ]
 ```
 
+## Example Repos
+
+- [WhisperKit](https://github.com/argmaxinc/WhisperKit): A Swift Package for state-of-the-art speech-to-text systems from [Argmax](https://github.com/argmaxinc)
+- [MLX Swift Examples](https://github.com/ml-explore/mlx-swift-examples): A Swift Package for integrating MLX models in Swift apps.
+
+## Supported Models
+
+This package has been tested with autoregressive language models such as:
+
+- GPT, GPT-Neox, GPT-J.
+- SantaCoder.
+- StarCoder.
+- Falcon.
+- Llama 2.
+
+Encoder-decoder models such as T5 and Flan are currently _not supported_. They are high up in our [priority list](#roadmap).
+
 ## Other Tools
 
 - [`swift-chat`](https://github.com/huggingface/swift-chat), a simple app demonstrating how to use this package.
 - [`exporters`](https://github.com/huggingface/exporters), a Core ML conversion package for transformers models, based on Apple's [`coremltools`](https://github.com/apple/coremltools).
 - [`transformers-to-coreml`](https://huggingface.co/spaces/coreml-projects/transformers-to-coreml), a no-code Core ML conversion tool built on `exporters`.
 
+## Contributing 
+
+Swift Transformers is a community project and we welcome contributions. Please
+check out [Issues](https://github.com/huggingface/swift-transformers/issues)
+tagged with `good first issue` if you are looking for a place to start!
```

Comment:
Cool! Do we have any? Happy to create a few later!

Reply:
There is a couple!